healthcare/dataset_test.go

```diff
@@ -50,9 +51,15 @@ func TestDataset(t *testing.T) {
 		})
 	}
 
-	if err := createDataset(buf, tc.ProjectID, location, datasetID); err != nil {
-		t.Fatalf("createDataset got err: %v", err)
+	{
+		ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+		if err := createDataset(ctx, buf, tc.ProjectID, location, datasetID); err != nil {
+			cancel()
```

Comment:
I don't believe cancelling explicitly is necessary when you use a timeout... @enocom can you sanity check me on that?